app/src/main/java/com/nervousfish/nervousfish/data_objects/tap/ATapData.java

```diff
@@ -33,4 +33,29 @@ public Timestamp getTimestamp() {
         // Defensive copying
         return new Timestamp(this.timestamp.getTime());
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        
+        final ATapData ts = (ATapData) o;
+
+        return this.timestamp == null ? ts.timestamp == null : this.timestamp.equals(ts.timestamp);
```

Comment:
Could you make this `equals` more like the one [in multicontacts](https://github.com/ericcornelissen/NervousFish/pull/190/files#diff-149fcabef3cefb35343ebbd60e7f63aeR38) 😇